codegenerator/model/api.go

```diff
@@ -147,7 +147,7 @@ type ` + api.apiDef.Name + ` tcclient.ConnectionData
 	content += "//  if err != nil {\n"
 	content += "//  	// handle errors...\n"
 	content += "//  }\n"
-	content += "func New(credentials tcclient.Credentials) *" + api.apiDef.Name + " {\n"
+	content += "func New(credentials *tcclient.Credentials) *" + api.apiDef.Name + " {\n"
```

Comment:
These three changes in this file are the ones that cause all the changes to the generated code (index, awsprovisioner, queue, github, scheduler, ....).